### ISSUE TITLE
Possibility to add documents on receive order

### ISSUE THREAD

**Issue by patrickue** [foodcoops/foodsoft#549]
We often get delivery documents (delivery note) with each order, and they sometimes contain handwritten or other notes which are good to keep with the order. So adding them in the receive order dialog as pdf/jpg would be great.

**Comment by wvengen** [comment]
Does it make sense to use the finance > invoices feature for this (which can be linked to an order or delivery), or is this something else than an invoice?

**Comment by patrickue** [comment]
Yes and no. This is not an invoice. More like a delivery note, stating which goods were delivered. Like the note you get in an online order with the package, stating which items were packed into the cardboard box. And you can check if everything arrived in good condition, or if the a delivering something late (or not at all).

In our FoodCoop, this list get checked back with the actual food delivered. If there is some discrepancy, people make a note on the list whether something was missing or delivered extra. It a kind of redundancy to the "Receive" of an order in the foodsoft, where you state what came and what didn't. This document is often handy to check on problems with the invoice and to clear up questions between finance and supplier.

So adding them as an "Invoice" could be a workaround for us, yes. But it doesn't state a amount of €, just a list of goods delivered.

**Comment by wvengen** [comment]
Ah, I get the point. I think it indeed makes sense to include this for an order.
A link to the document could be shown in the receive screen.

A future improvement may be to parse the PDF/whatever and pre-fill the receive screen (similar to product import file formats).
